bpf/lib/lb.h

```diff
@@ -880,6 +878,9 @@ static __always_inline int lb6_local(const void *map, struct __ctx_buff *ctx,
 			      SCOPE_REVERSE, CT_ENTRY_SVC, state, &monitor);
 	switch (ret) {
 	case CT_NEW:
+		if (unlikely(svc->count == 0))
+			goto no_service;
+
```

Comment:
@borkmann with Session-Affinity, maybe this should even move lower? I'd need to understand more how Session-Affinity deals with stale backends ...

(strictly as follow-up of course)

Reply:
Technically this is a /new/ connection, and given there is no backend anymore for the service, I think dropping is the right answer. Otherwise we're draining new traffic but in this case we make an exception and let a new connection to be established. I think as-is is fine imho.